Installation/include/CGAL/config.h

```diff
@@ -36,11 +36,6 @@
 #  define WIN64
 #endif
 
-#ifdef _MSC_VER
-#define _SILENCE_CXX17_ALLOCATOR_VOID_DEPRECATION_WARNING 1
-#define _SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING 1
-#endif
-
```

Comment:
Are you sure about that?

Reply:
If I remember well, it was not for CGAL code, but for Boost. You will probably see that in the testsuite results.